application/helpers/admin/htmleditor_helper.php

```diff
@@ -208,12 +208,91 @@ function getPopupEditor($fieldtype, $fieldname, $fieldtext, $surveyID = null, $g
     } else {
         $class = "editorLink input-group-text";
     }
-    $htmlcode .= ""
-    . "<a href=\"javascript:start_popup_editor('" . $fieldname . "','" . addslashes(htmlspecialchars_decode((string) $fieldtext, ENT_QUOTES)) . "','" . $surveyID . "','" . $gID . "','" . $qID . "','" . $fieldtype . "','" . $action . "')\" id='" . $fieldname . "_ctrl' class='{$class} btn btn-outline-secondary btn-xs'>\n"
-    . "\t<i class='ri-pencil-fill btneditanswerena' id='" . $fieldname . "_popupctrlena' data-bs-toggle='tooltip' data-bs-placement='bottom' title='" . gT("Start HTML editor in a popup window") . "'></i>"
-    . "\t<i class='ri-pencil-fill btneditanswerdis' id='" . $fieldname . "_popupctrldis'  style='display:none'  ></i>"
-    . "</a>\n";
-
+    /* @var string[] parameters of the editor url */
+    $editorurlparams = array(
```

Comment:
For new variables we decided for lowerCamelCase - please check out https://manual.limesurvey.org/Code_quality_guide#Variables

Reply:
Copy code from september 2022 …

Only in 6.X ? Right ?